src/components/Calculator.js

```diff
@@ -1,59 +1,51 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-// eslint-disable-next-line
-class Calculator extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      obj: {
-        total: '0',
-        next: '',
-        operation: '',
-      },
-    };
-    this.handleClick = this.handleClick.bind(this);
-  }
+const initial = {
+  total: '0',
+  next: '',
+  operation: '',
+};
 
-  handleClick = (obj, btnValue) => {
-    const newObj = calculate(obj, btnValue);
+const Calculator = () => {
```

Comment:
- [ ] When I use the modulus with 0 it throws an error, please handle this by displaying a message to the user that they cannot use modulus with 0

Reply:
Hi @GzimAsani thanks for your suggestion, can you explain to me, what do you mean with 0?

Reply:
When I do for example 8 % 0 or anything with 0 it throws an error. Sry for the late reply

Reply:
Thank you for clarifying it, yes that's true I didn't notice yet! The error is only with % 0, I'll fix it!!